### PR TITLE
Fix minor typos (added to Run LAcheck after daligner)

### PR DESCRIPTION
**Pull request by pb-jchin**

### CODE REVIEW

File: falcon_kit/functional.py [hunk 1]
```diff
@@ -80,12 +80,14 @@ def LAsort_pair(line):
...
     result = {}
     for dali, pairs in dali2pairs.items():
-        sorts = [pair2sort[pair] for pair in sorted(pairs, key=lambda k: (
+        sorted_pairs = sorted( pairs,  key=lambda k: (
             (int(k[0][1:]) if k[0].startswith('.') else 0),
             (int(k[1][1:]) if k[1].startswith('.') else 0)
-        ))]
+        ))
+        sorts = [pair2sort[pair] for pair in sorted_pairs]
+        checks = ["LAcheck {0}.db L1{1}{2}.las || exit 1".format(db_prefix, p1, p2)  for p1, p2 in sorted_pairs]
```
Comment (pb-cdunn) [inline falcon_kit/functional.py]: I think we already run with `set -x`, so `||exit ` should be superfluous. I'll delete it later.

Comment (pb-jchin) [inline falcon_kit/functional.py]: ok, thanks.

Comment (pb-jchin) [inline falcon_kit/functional.py]: one more note, `LAcheck` does not find all errors, but that is a different issue.  (I manually permute and delete the bytes in a `las` file, but LAcheck fails to find the corruption.  I have not checked what exactly `LAcheck` does.)

File: falcon_kit/util/system.py [hunk 2]
```diff
@@ -18,10 +18,10 @@ def only_these_symlinks(dir2paths):
         base2rel = {os.path.basename(path): os.path.relpath(path, d) for path in paths}
         assert len(base2rel) == len(bases), 'Non-unique basename in {}'.format(repr(paths))
         for existing_base in os.listdir(d):
-            existing_path = os.path.join(d, existing_basename)
+            existing_path = os.path.join(d, existing_base)
```
Comment (pb-cdunn) [inline falcon_kit/util/system.py]: Already fixed on branch `master`. If you have time, you can update this PR like this:
```sh
git pull --rebase
git push -f pb-chin my-branch:minor_type_fix
```

Comment (pb-cdunn) [inline falcon_kit/util/system.py]: Actually, this whole PR is a tiny change, but with several commits. Now is a good time to learn how to squash with interactive rebase:
```sh
git checkout -B minor_type_fix
git pull --rebase origin/master
git rebase -i origin/master
```
Then, in your editor, change `pick` to `squash`. But keep the **first** `pick`. Then save and exit. Your squashed commits will become part of the the picked commit.

Try it! If you screw up, no problem. Everything can be undone in git!

**Comment by pb-cdunn** [comment]
Well, maybe you can learn`git rebase -i` some other time. I've squashed for you and created a new pull-request.

Btw, if we usually push into feature branches in the PB repo, then we can update each other's pull-requests, which is an easy way to collaborate.

**Pull request closed without merging**